ensemble/volren/volume_axes.py

```diff
@@ -20,6 +20,16 @@ class VolumeAxes(ABCVolumeSceneMember):
     # Which axes should have a scale shown?
     visible_axis_scales = Tuple(Bool, Bool, Bool)
 
+    # Axes titles.
+    axis_titles = Tuple(Str, Str, Str)
```

Comment:
Hmm; that's a shame.

What's the failure mode when someone supplies a Unicode string for an axis title?